### PR TITLE
baby-calendar.jp

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/112288

<details><summary>Screenshot:</summary>

Снимок экрана 2022-03-23 в 20 13 33

Снимок экрана 2022-03-23 в 20 14 20

</details><br/>

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -39,6 +39,7 @@ financial-field.com,spaia.jp,yomiuri.co.jp,fujinkoron.jp,mikle.jp,onayamifree.co
 !
 !
 cosme.net##.atcosme-HeaderAdContent
+baby-calendar.jp###sp
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:42]: Where this is used?

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -13,7 +13,7 @@
 ! ##.pr
 ceron.jp,kids.gakken.co.jp,5chan.jp,toyokeizai.net,twpf.jp,chunichi.co.jp,tokyo-np.co.jp,db.msin.jp,ladiesfashion.jp,minpo.jp##.pr
 ! ##.bnr
-npn.co.jp,afpbb.com,2ch.pet,hamakei.com,keizai.biz,minkei.net,shibukei.com##.bnr
+baby-calendar.jp,npn.co.jp,afpbb.com,2ch.pet,hamakei.com,keizai.biz,minkei.net,shibukei.com##.bnr
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:16]: Hmm, blocks app banner on `https://baby-calendar.jp/recipe/5739` too on mobile. @zloyden What do you think? I guess it's acceptable.

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 3]
```diff
@@ -24,6 +24,10 @@
 !
 hk01.com##div[id^="ig-"][data-title$="app download"]
 tiktok.com##.footer-guide
+||baby-calendar.jp/images/default_web/banner_side_app.png
+baby-calendar.jp##a[href^="https://itunes.apple.com/jp/app"]
+baby-calendar.jp##a[href^="https://play.google.com/store/apps"]
+baby-calendar.jp##.box-side-bana
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/mobile-app.txt:30]: Add `baby-calendar.jp##.floating_btn` https://github.com/AdguardTeam/AdguardFilters/issues/112288#issuecomment-1062171751 (need scrolling a bit)
and
`||baby-calendar.jp/img/common/app/news-header1.png`
(`[https://baby-calendar.jp/smilenews/detail/24890](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fbaby-calendar.jp%2Fsmilenews%2Fdetail%2F24890)`)

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Add `baby-calendar.jp##div[id^="JP_BABY_sp_all_"]` (not always covered by `##.bnr`) and `baby-calendar.jp##.linkArea` (on `https://baby-calendar.jp/article/11195`) to Mobile filter

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 3 above)
Comment (Yuki2718) [reply AnnoyancesFilter/sections/mobile-app.txt:30]: Or maybe the previous rule `||baby-calendar.jp/img/common/app/header_` (I moved to mobile_app) can be trimmed down to `||baby-calendar.jp/img/common/app/`.

File: JapaneseFilter/sections/css_extended.txt [hunk 4]
```diff
@@ -102,6 +102,7 @@ otonasalone.jp,sumaiweb.jp#?#._popIn_infinite_page > a._popIn_recommend_article_
 !
 !
 financial-field.com#?#.entry_main_body > p.contentText:contains(【PR】)
+baby-calendar.jp#?#.center-xs:has(> div[id^="JP_BABY_resp"])
```
Comment (Yuki2718) [inline JapaneseFilter/sections/css_extended.txt:105]: Where?

Comment (Yuki2718) [reply JapaneseFilter/sections/css_extended.txt:105]: Needing `baby-calendar.jp###JP_BABY_resp_all_footer_all_right_2nd` too.

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (zloyden) [reply JapaneseFilter/sections/specific.txt:16]: Can you show a screenshot? I don't see it on my end.

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (jellizaveta) [reply JapaneseFilter/sections/specific.txt:42]: Unfortunately, I can't find `baby-calendar.jp##div[id^="JP_BABY_sp_all_"]` and `baby-calendar.jp##.linkArea` on this page. Can I ask you to provide a screenshot?

Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:72]: Found false positive by `##.linkArea` so no need.
<details>
<summary>Screenshots</summary>

baby1

baby2

</details>

Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:72]: Directly added `baby-calendar.jp##div[id^="JP_BABY_sp_all_"]`

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 3 above)
Comment (Yuki2718) [reply AnnoyancesFilter/sections/mobile-app.txt:41]: Directly edited

File: JapaneseFilter/sections/css_extended.txt (hunk 4 above)
Comment (Alex-302) [reply JapaneseFilter/sections/css_extended.txt:106]: Ok.

**Pull request merged**